plugin/commentary.vim

```diff
@@ -44,6 +44,14 @@ function! s:go(...) abort
     endif
   endfor
 
+  " Insert comment characters into beginning of line if
```

Comment:
Mention this in the `doc/` file near the `b:commentary_format` explanation, not a comment.

Reply:
Removed the comment, added doc